Compatibility with older livestatus releases

llange opened the pull request:
Detect peer support for the following columns, instead of requesting them unconditionally:
* services.check_freshness
* hosts.event_handler
* (services|hosts).staleness

Resolves #116

sni commented on lmd/objects.go:
```diff
@@ -275,7 +275,7 @@ func NewHostsTable() (t *Table) {
 	t.AddColumn("custom_variable_names", Static, StringListCol, "A list of the names of all custom variables")
 	t.AddColumn("custom_variable_values", Dynamic, StringListCol, "A list of the values of the custom variables")
 	t.AddColumn("display_name", Static, StringCol, "Optional display name of the host")
-	t.AddColumn("event_handler", Static, StringCol, "Naemon command used as event handler")
```
could you please move this to the end of this function to the other optional columns.

llange replied:
Done

sni reviewed (changes requested):
looks great, just a small minor change request. The optional columns are grouped after the standard columns.

llange commented:
I just added the necessary changes to make the `citest` pass.

I just don't understand exactly [the change I had to apply to `table.go`](https://github.com/sni/lmd/pull/117/files#diff-c043eb73e021ee95a2d66292a612c06871095628c5f689d494830f49969ebf8eL208-R208).

Should I also add the other flags I added in this PR (`HasCheckFreshnessColumn` and `HasEventHandlerColumn`) ?
I doesn't change the output of the tests.

sni commented:
I'll have a look. The change to table.go should not be required. I need to rework the example/test data, that's a mess right now.

sni commented:
thanks, i reworked optional column handling and test data and manually applied your pull request while doing that.
